index.js

```diff
@@ -1,282 +1,33 @@
-const express = require("express");
-const config = require("./config.js");
-const axios = require("axios");
-const Discord = require("discord.js");
-
-const admin = require("firebase-admin");
-let serviceAccount;
-if (process.env.FIREBASE_SERVICE_ACCOUNT) {
-  serviceAccount = JSON.parse(process.env.FIREBASE_SERVICE_ACCOUNT);
-} else {
-  serviceAccount = require("./yu-gi-oh-inventory-firebase-adminsdk-z8h62-7925253eaa.json");
-}
-
-admin.initializeApp({
-  credential: admin.credential.cert(serviceAccount),
-  databaseURL: "https://yu-gi-oh-inventory.firebaseio.com",
-});
-
-const db = admin.database();
+import express from 'express';
```

Comment:
some of these files using double quotes and some are single quotes, lint should be catching that 🤔

Reply:
I didn't save all the files a second time after adding prettier to the eslint extends array. Prettier changes all the single quotes back to doubles when I save now.